src/Form/FieldSelect.js

```diff
@@ -0,0 +1,196 @@
+        classSet[className] = classNames(
+          classes[className],
+          this.props[className]
+        );
+        return classSet;
+    }.bind(this), classes);
+
+    if (defaultValue == null) {
+      defaultValue = dropdownItems[0];
+    }
+
+    if (typeof defaultValue === 'string'){
+      defaultValue = {
+        html: defaultValue,
+        id: defaultValue.toLowerCase()
+      };
+    }
+
+    return (
+      <Dropdown {...classes}
+        initialID={defaultValue.id}
+        items={dropdownItems}
+        onItemSelection={this.handleChange.bind(this)}
+        persistentID={persistentID}
+        transition={true} />
+    );
+  }
+
+  getErrorMsg() {
+    let errorMsg = null;
+    let {props} = this;
+
+    if (this.hasError()) {
+      errorMsg = (
+        <p className={props.helpBlockClass}>
+          {props.validationError[props.name]}
+        </p>
+      );
+    }
+
+    return errorMsg;
+  }
+
+  getLabel() {
+    let {label, showLabel} = this.props;
+
+    if (!showLabel) {
+      return null;
+    }
+
+    if (typeof showLabel === 'string') {
+      label = showLabel;
+    }
+
+    if (typeof showLabel !== 'string' && showLabel !== true) {
+      return showLabel;
+    }
+
+    return (
+      <label>{label}</label>
+    );
+  }
+
+  render() {
+    let {
+      columnWidth, formGroupClass, formElementClass, formGroupErrorClass,
+      options
+    } = this.props;
+
+    options = options.map(function (option) {
+      if (typeof option === 'string') {
+        return {
+          html: option,
+          id: option.toLowerCase()
+        };
+      }
+
+      return option;
+    });
+
+    let classes = classNames(
+      formGroupClass,
+      {
+        [formGroupErrorClass]: this.hasError()
+      }
+    );
+
+    let rowClass = classNames(
+      `column-${columnWidth}`,
+      formElementClass
+    );
+
+    return (
+      <div className={rowClass}>
+        <div className={classes}>
+          {this.getLabel()}
+          {this.getDropDown(options)}
+          {this.getErrorMsg()}
+        </div>
+      </div>
+    );
+  }
+}
+
+FieldSelect.defaultProps = {
+  columnWidth: 12,
+  formElementClass: 'form-row-element'
+};
+
+FieldSelect.propTypes = {
+  // Optional number of columns to take up of the grid
+  columnWidth: React.PropTypes.number.isRequired,
+
+  // Name of the field property
+  // (usually passed down from form definition)
+  name: React.PropTypes.string.isRequired,
+  // Optional boolean, string, or react node.
+  // If boolean: true - shows name as label; false - shows nothing.
+  // If string: shows string as label.
+  // If node: returns the node as the label.
+  showLabel: React.PropTypes.oneOfType([
+    React.PropTypes.node,
+    React.PropTypes.string,
+    React.PropTypes.bool
+  ]),
+
+  // Function to handle events like 'change', 'blur', 'focus', etc on the field
+  // (usually passed down from form definition)
+  handleEvent: React.PropTypes.func,
```

Comment:
You don't have this set to required nor do you have a default, and you also don't check to make sure it's a function before invoking. Let's do one of those options to ensure there's no exceptions!